src/Angor/Avalonia/Angor.Contexts.Funding/Investor/IInvestmentAppService.cs

```diff
@@ -8,8 +8,8 @@ namespace Angor.Contexts.Funding.Investor;
 
 public interface IInvestmentAppService
```

Comment:
Jose, perhaps the app service interfaces should be the only things in the project that are public, the rest can be internal for the testing project to use

Reply:
You're absolutely right! Let's do that in a separate PS. I'll create a card so I don't forget it.